tests/problem3.test.js

```diff
@@ -23,36 +23,49 @@
  * });
  */
 
-const p = require('../utils.js');
+const p = require("../utils.js");
 
-describe('problem3', () => {
-    it('resolves to first resolved value', async () => {
-        const result = await first([
-            p.resolveWith(1).after(10),
-            p.resolveWith(5).after(50)
-        ]);
+const first = async promises => {
```

Comment:
Niestety popełnilliśmy błędy w testach do tego zadania. To nie jest dobre rozwiązanie. Polecam domerdżować sobie mastera i zastanowić się, czemu testy już nie przechodzą. ;)